Package.swift

```diff
@@ -19,7 +19,7 @@
 import PackageDescription
 import class Foundation.ProcessInfo
 
-let firebaseVersion = "9.3.0"
+let firebaseVersion = "9.4.0"
```

Comment:
Do we have a process in place to ensure Analytics is changed to 9.4.0 before release since it can't be changed here until the [GoogleAppMeasurement](https://github.com/google/GoogleAppMeasurement.git) repo is updated?

Reply:
This looks like a regression from #9829. The GoogleAppMeasurement versions should not be updated every release. cc; @ncooke3

Reply:
Disregard my previous statement. The GoogleAppMeasurement podspec versions should be updated, but not the SPM versions or pod archives. The reason for the CI failures is the podspecs need to be pushed to SpecsStaging

Reply:
Pods are currently being published to SpecsStaging.